lib/syntax.ml

```diff
@@ -178,7 +185,12 @@ let resolve_aggregations =
     | ResInparam _| ResChoices (_, _)
     | ResAggValue _ | ResInTupleList _ -> res_expr
   in
-  handle ~is_agg:false  
+  handle ~is_agg:false 
+  
+let resolve_ambiguous_columns all_schema final_schema = 
+  List.filter (fun s1 ->
```

Comment:
filter only applies to all_schema - make it more clear

Reply:
this was not addressed
i mean `let l = List.filter ... all_schema in l @ final_schema`

Reply:
Ah, I thought you meant to me to name it more clearly. Made more clear ( it to be not to looked that `@` applies to final_shema and all_schema before filter applies to all_schema)